test/test_plugins.py

```diff
@@ -401,6 +402,155 @@ def dummy9(self, **kwargs):
         plugins.send('event9', foo=5)
 
 
+class PromptChoicesTest(TerminalImportSessionSetup, unittest.TestCase,
+                        ImportHelper, TestHelper):
+    def setUp(self):
+        self.setup_plugin_loader()
+        self.setup_beets()
+        self._create_import_dir(3)
+        self._setup_import_session()
+        self.matcher = AutotagStub().install()
+        # keep track of ui.input_option() calls
+        self.input_options_patcher = patch('beets.ui.input_options',
+                                           side_effect=ui.input_options)
+        self.mock_input_options = self.input_options_patcher.start()
+
+    def tearDown(self):
+        self.input_options_patcher.stop()
+        self.teardown_plugin_loader()
+        self.teardown_beets()
+
+    def test_plugin_choices_in_ui_input_options_album(self):
```

Comment:
This one and `test_plugin_choices_in_ui_input_options_singleton` make assertions on the parameters passed to `ui.input_options` as a way to verify that the options are indeed presented to the user - I felt this was cleaner than directly comparing `stdout`.